src/Speech/V1/Gapic/SpeechGapicClient.php

```diff
@@ -500,6 +506,13 @@ public function longRunningRecognize($config, $audio, $optionalArgs = [])
      */
     public function streamingRecognize($optionalArgs = [])
     {
+        if (array_key_exists('timeoutMillis', $optionalArgs)) {
```

Comment:
Are we not able to normalize the settings used for streaming methods?

Reply:
We don't support retrying on streaming methods. So I think that it makes sense to document and support only a `timeoutMillis` settings. However, it is still possible to specify 'retrySettings', and that will be used. But I don't think we want to document it, because it will cause an error if retriesEnabled is not false.